sdk/digitaltwins/Azure.DigitalTwins.Core/samples/Readme.md

```diff
@@ -322,7 +322,8 @@ string string1 = new AdtQueryBuilder()
     .Select("*")
     .From(AdtCollection.DigitalTwins)
     .Build()
-    .ToString();
+    .Stringify();
```

Comment:
I think we adopted the method name from the origin library. I don't love it, though. What about `GetQueryText()` or something similar?

Reply:
`Stringify` is a made-up name by Azad. I agree that `GetQueryText()` is a much better name though.